spacy/lang/punctuation.py

```diff
@@ -44,3 +44,16 @@
         r"(?<=[{a}0-9])[:<>=/](?=[{a}])".format(a=ALPHA),
     ]
 )
+
+
+# Some languages written with the Cyrillic alphabet permit the use of diacritics
+# to mark stressed syllables in words where stress is distinctive. Such languages
+# should use 'CYRILLIC_TOKENIZER_INFIXES' in place of 'TOKENIZER_INFIXES'.
+CYRILLIC_TOKENIZER_INFIXES = list(TOKENIZER_INFIXES) + [
```

Comment:
The naming could be more general here, right? It could be something like "COMBINING_DIACRITICS_TOKENIZER_INFIXES`?